perception/multi_object_tracker/src/multi_object_tracker_core.cpp

```diff
@@ -45,6 +45,13 @@ boost::optional<geometry_msgs::msg::Transform> getTransformAnonymous(
   const std::string & target_frame_id, const rclcpp::Time & time)
 {
   try {
+    // check if the frames are ready
+    std::string errstr;  // This argument prevents error msg from being displayed in the terminal.
+    if (!tf_buffer.canTransform(
+          target_frame_id, source_frame_id, tf2::TimePointZero, tf2::Duration::zero(), &errstr)) {
```

Comment:
```suggestion
          target_frame_id, source_frame_id, tf2::TimePointZero, rclcpp::Duration::from_seconds(0.5), &errstr)) {
```

Reply:
@yukkysaito Jst to confirm. This `canTransform` just checks if the latest data exist, so once the frames are set it returns always `true`. Do you think the timeout is needed?

Reply:
reason : https://github.com/autowarefoundation/autoware.universe/pull/2511/files#diff-2dd63aedc3acfbc0b6da8fd536d829ac3754d1e4743863040083a3ebe4723666R58

maybe better to change 58 line is `tf2::Duration::zero()`

Reply:
I see :+1: 
When we re-initialize the self position,  maybe the code does not work as intended.